source/userguide/auto_annotator.rst

```diff
@@ -507,6 +723,6 @@ Getting Custom Parameters from the Config
 		... 
 	}
 
-:class:`SpacyAnnotator` checks if :attr:`spacy_model` exists in the config, and if it doesn't, it will use the default value of "en_core_web_lg".
+If a :class:`SpacyAnnotator` is created using the command-line with will use the value for :attr:`spacy_model_size` that exists in the config during instantiation.
```

Comment:
Nit: 
```suggestion
If a :class:`SpacyAnnotator` is created using the command-line, it will use the value for :attr:`spacy_model_size` that exists in the config during instantiation.
```